build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1530,6 +1530,10 @@ message DigestFunction {
 
     // The SHA-512 digest function.
     SHA512 = 6;
+
+    // Murmur3 digest function.
+    // See https://github.com/aappleby/smhasher/wiki/MurmurHash3 .
+    Murmur3 = 7;
```

Comment:
Aren’t enum values supposed to be capitalized? Which variant of Murmur3 is being used? Wikipedia says this:

> The current version is MurmurHash3,[4][5] which yields a 32-bit or 128-bit hash value. When using 128-bits, the x86 and x64 versions do not produce the same values, as the algorithms are optimized for their respective platforms. MurmurHash3 was released alongside SMHasher—a hash function test suite.

Reply:
Latest update:
- Capitalized enum per multiple comments.
- Clarified the variant of Murmur in use.
- Highlighted the non-crypto nature of the hash.